vertx-platform/src/main/java/org/vertx/java/platform/configuration/ConfigurationUtils.java

```diff
@@ -0,0 +1,44 @@
+package org.vertx.java.platform.configuration;
+
+import org.vertx.java.core.json.DecodeException;
+import org.vertx.java.core.json.JsonObject;
+import org.vertx.java.core.logging.Logger;
+import org.vertx.java.core.logging.impl.LoggerFactory;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+
+/**
+ *
+ */
+public class ConfigurationUtils {
+	private static final Logger log = LoggerFactory.getLogger(ConfigurationUtils.class);
```

Comment:
If this is a constant (static final) variable and we want to be very strict regarding naming conventions, this should be called LOG